htdocs/install/sql/mysql.structure.sql

```diff
@@ -491,6 +491,7 @@ CREATE TABLE tplsource (
 
 CREATE TABLE users (
   `uid` mediumint(8) unsigned NOT NULL auto_increment,
+  `hash_uid` varchar(64) NOT NULL default '',
```

Comment:
If we use DB field, it should be unique; just add similar line like 'login_name'